src/library/treeitemmodel.cpp

```diff
@@ -193,3 +193,9 @@ TreeItem* TreeItemModel::getItem(const QModelIndex &index) const {
     }
     return m_pRootItem;
 }
+
+void TreeItemModel::triggerRepaint() {
+    //todo: find a nicer way to do this. Works as it should but kind of hackish
```

Comment:
Yes, i tried emit dataChanged() with various parameters. I couldn't get it to work with that. From that point on, I was unable to repaint only some items so I stick with repainting the whole tree.